graphql/error/graphql_error.py

```diff
@@ -166,7 +166,7 @@ def __eq__(self, other):
         )
 
     def __hash__(self):
-        return hash(self.original_error)
+        return super(GraphQLError, self).__hash__()
```

Comment:
That should be just `super().__hash__()` in Python 3. Or even remove the `def` and simply set  `__hash__ = Exception.__hash__` here.